logmiddleware.go

```diff
@@ -43,14 +43,19 @@ type Response struct {
 }
 
 type Host struct {
-	Hostname string `json:"hostname,omitempty"`
-	IP       string `json:"ip,omitempty"`
+	Hostname      string `json:"hostname,omitempty"`
+	ForwardedHost string `json:"forwardedHost,omitempty"`
+	IP            string `json:"ip,omitempty"`
 }
 
 type URL struct {
 	Path string `json:"path,omitempty"`
 }
 
+func getHostname(host string) string {
```

Comment:
Please add a test verifying the behaviour with and without the port.

Reply:
ok